config/blade-fontawesome.php

```diff
@@ -149,6 +149,21 @@
 
     ],
 
+    'sharp-duotone-solid' => [
+    
+        'prefix' => 'fasds',
```

Comment:
The prefixes seem to have never worked for the ones with dashes in, I've merged #77 and opened #85 to see if I can get around not having a prefix separator with a colon.

Would you be able to test if this works for you? 🤔 Basically by changing this from `fasds` to `fas:ds` (and then using it via Blade).

I've tested it locally, but I don't have Pro anymore, so can't really test easily.

Reply:
I tried adding sharp-duotone-solid using `fas:ds`, and it worked.